2021-04-29 20:26:38,145 INFO     ---------------------------------------------------------------------------------------------
2021-04-29 20:26:38,146 INFO     Geo: cqd
2021-04-29 20:26:38,146 INFO     Data Path: data/FB15k-237-betae
2021-04-29 20:26:38,146 INFO     #entity: 14505
2021-04-29 20:26:38,146 INFO     #relation: 474
2021-04-29 20:26:38,146 INFO     #max steps: 1000
2021-04-29 20:26:38,146 INFO     Evaluate unoins using: DNF
2021-04-29 20:26:38,146 INFO     loading data
2021-04-29 20:27:25,552 INFO     Training info:
2021-04-29 20:27:25,559 INFO     Validation info:
2021-04-29 20:27:25,559 INFO     inp: 5000
2021-04-29 20:27:25,563 INFO     Test info:
2021-04-29 20:27:25,563 INFO     inp: 5000
2021-04-29 20:27:25,903 INFO     Model Parameter Configuration:
2021-04-29 20:27:25,904 INFO     Parameter embeddings.0.weight: torch.Size([14505, 2000]), require_grad = True
2021-04-29 20:27:25,904 INFO     Parameter embeddings.1.weight: torch.Size([474, 2000]), require_grad = True
2021-04-29 20:27:25,904 INFO     Parameter Number: 29958000
2021-04-29 20:27:30,910 INFO     Loading checkpoint models/fb15k-237-betae...
2021-04-29 20:27:31,428 INFO     tasks = inp
2021-04-29 20:27:31,429 INFO     init_step = 99999
2021-04-29 20:27:31,429 INFO     batch_size = 1000
2021-04-29 20:27:31,429 INFO     hidden_dim = 1000
2021-04-29 20:27:31,429 INFO     gamma = 12.000000
2021-04-29 20:27:31,429 INFO     Evaluating on Valid Dataset...
  0%|          | 0/5000 [00:00<?, ?it/s]
main.py --do_valid --do_test --data_path data/FB15k-237-betae -n 1 -b 1000 -d 1000 -lr 0.1 --disable_warmup --max_steps 1000 --cpu_num 0 --geo cqd --valid_steps 20 --tasks inp --print_on_screen --test_batch_size 1 --optimizer Adagrad --reg_weight 0.05 --log_steps 5 --checkpoint_path models/fb15k-237-betae --cqd d2 --cqd-t-norm min --cqd-normalize --cqd-sigmoid --cqd-k 1 --no-save --cuda
overwritting args.save_path
logging to models/fb15k-237-betae
99999
Traceback (most recent call last):
  File "main.py", line 508, in <module>
    main(parse_args())
  File "main.py", line 497, in main
    valid_all_metrics = evaluate(model, valid_easy_answers, valid_hard_answers, args, valid_dataloader, query_name_dict, 'Valid', step, writer)
  File "main.py", line 165, in evaluate
    metrics = KGReasoning.test_step(model, tp_answers, fn_answers, args, dataloader, query_name_dict)
  File "/home/pminervi/workspace/KGReasoning/models.py", line 657, in test_step
    argsort = torch.argsort(negative_logit, dim=1, descending=True)
TypeError: argsort() received an invalid combination of arguments - got (NoneType, descending=bool, dim=int), but expected one of:
 * (Tensor input, name dim, bool descending)
      didn't match because some of the arguments have invalid types: (!NoneType!, !dim=int!, descending=bool)
 * (Tensor input, int dim, bool descending)
      didn't match because some of the arguments have invalid types: (!NoneType!, dim=int, descending=bool)

